README.md

```diff
@@ -3,3 +3,44 @@
 [![Build Status](https://travis-ci.org/foss-for-synopsys-dwc-arc-processors/arc_labs.svg?branch=master)](https://travis-ci.org/foss-for-synopsys-dwc-arc-processors/arc_labs)
 
 The labs of ARC university courses
+
+## Build documentation locally
+
+### Installation
+
+#### Prerequisite
+
+Install Sphinx, either from a distribution package or from PyPI. And install it fromPyPI is the easier way. So you should install **Python** first.
+
+#### Install 
+
+	pip install Sphinx 
+    pip install breathe
+    pip install recommonmark 
+    pip install sphinx_rtd_theme 
+
+### Build the documentation
+1. Sphinx comes with a script called sphinx-quickstart that sets up a source directory and creates a default conf.py with the most useful configuration values from a few questions it asks you. Just run
+
+		sphinx-quickstart
+
+	and answer its questions.
```

Comment:
We already have a makefile in doc, no need to run this sphinx-quickstart, I think.